src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -175,14 +175,24 @@ void AppCommandlineArgs::_buildParser()
     //   -M,--maximized: Maximizes the window on launch
     //   -F,--fullscreen: Fullscreens the window on launch
     auto maximizedCallback = [this](int64_t /*count*/) {
-        _launchMode = winrt::Microsoft::Terminal::Settings::Model::LaunchMode::MaximizedMode;
+        _launchMode = (_launchMode.has_value() && _launchMode.value() == winrt::Microsoft::Terminal::Settings::Model::LaunchMode::FocusMode) ?

```

Comment:
You can drop the `winrt::Microsoft::Terminal::Settings::Model::` prefix, because the file [already imports the namespace](https://github.com/microsoft/terminal/blob/ad03ae0ab889f81b76ed339619e739cd5c9c468a/src/cascadia/TerminalApp/AppCommandlineArgs.cpp#L10). 👍

Reply:
Thanks! Done!